src/compile.js

```diff
@@ -235,22 +235,49 @@ export class Transformer extends BasisTransformer {
         v0.forEach((v) => {
           if (Number.isInteger(v)) {
             attrs.push(`border-${v}`);
-          } else {
-            attrs.push(v);
-          }
-          if (v == "red") {
+          } else if (v == "slate") {
```

Comment:
@dyerjesse After you make the changes mentioned below, you can consolidate all the color cases into one. Do it by defining a list of colors somewhere at the global level (outside of the Transformer class) like this:
```
const colors = ['slate', 'gray', 'zinc', ...]
```
And then replace all of the color conditions with one like this:
```
} else if (colors.includes(v)) {
  attrs.push(`border-${v}`);
}
```